doc/src/quickstart/index.rst

```diff
@@ -14,6 +14,24 @@ The first step is to `download VTR <https://verilogtorouting.org/download/>`_ an
 
 .. note:: Developers planning to modify VTR should clone the `VTR git repository <https://github.com/verilog-to-routing/vtr-verilog-to-routing/>`_.
 
+
+Environment Setup
+-----------------
+VTR requires several system packages and Python packages to build and run the flow.  You can install the required system packages using the following command (this works on Ubuntu 18.04 and 20.04, but you may require different packages on other Linux distributions):
+
+.. code-block:: bash
+
+    > ./install_apt_packages.sh
+
+Then, to install the required Python packages within a new virtual environment:
+
+.. code-block:: bash
+
+    > make env
+    > source .venv/bin/activate
+    > pip install -r requirements.txt
```

Comment:
Should make env run the next two commands too (fewer commands to run)?

Reply:
Or are you trying to show them how to do this system wide or in a virtual envronment?  If so, suggest putting a command on the two virtual environment lines.

make env    # optional: install python virtual environment
source .venv/bin/activate   # optional: activate python virtual environment
pip install -r requirements.txt   # install python packages (in virtual environment if prior commands run, system wide otherwise)

Reply:
Can't collapse commands as noted above, but I did add the comments.  Good idea.